backend/AAiT-backend-group-12/domain/domain.go

```diff
@@ -0,0 +1,40 @@
+package domain
+
+import (
+	"time"
+)
+
+type Blog struct {
+	ID         string
+	Title      string
+	Content    string
+	UserID     string
+	Tags       []string
+	CreatedAt  time.Time
+	UpdatedAt  time.Time
+	ViewCount  uint
+	LikedBy    []User
```

Comment:
I think this should be slice of string since we're only storing either the user's username or userID.

Reply:
noted.